packages/actor-rdf-resolve-quad-pattern-string-source/lib/ActorRdfResolveQuadPatternStringSource.ts

```diff
@@ -0,0 +1,92 @@
+import type { MediatorRdfParseHandle } from '@comunica/bus-rdf-parse';
+import { getContextSource
+  , ActorRdfResolveQuadPattern } from '@comunica/bus-rdf-resolve-quad-pattern';
+import type { IActionRdfResolveQuadPattern, IActorRdfResolveQuadPatternArgs, IActorRdfResolveQuadPatternOutput,
+  MediatorRdfResolveQuadPattern } from '@comunica/bus-rdf-resolve-quad-pattern';
+import { KeysRdfResolveQuadPattern } from '@comunica/context-entries';
+import type { IActorTest } from '@comunica/core';
+import type { IDataSource, ISerializeDataSource } from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import { storeStream } from 'rdf-store-stream';
+const streamifyString = require('streamify-string');
+
+/**
+ * A comunica RDF Resolve Quad Pattern String Source RDF Resolve Quad Pattern Actor.
+ */
+export class ActorRdfResolveQuadPatternStringSource extends ActorRdfResolveQuadPattern {
+  private readonly sourceType = 'stringSource';
+  private readonly mediatorRdfParse: MediatorRdfParseHandle;
+  private readonly mediatorRdfQuadPattern: MediatorRdfResolveQuadPattern;
+
+  public constructor(args: IActorRdfResolveQuadPatternStringSource) {
+    super(args);
+    this.mediatorRdfParse = args.mediatorRdfParse;
+    this.mediatorRdfQuadPattern = args.mediatorRdfQuadPattern;
+  }
+
+  public async test(action: IActionRdfResolveQuadPattern): Promise<IActorTest> {
+    const source = getContextSource(action.context);
+    if (!source) {
+      throw new Error(`Actor ${this.name} can only resolve quad pattern queries against a source.`);
+    }
+    const datasouce = source.valueOf();
+
+    if (!this.isStringSource(datasouce)) {
+      throw new Error(`Actor ${this.name} can only resolve stringSource quad pattern`);
+    }
+    return true;
+  }
+
+  public async run(action: IActionRdfResolveQuadPattern): Promise<IActorRdfResolveQuadPatternOutput> {
+    const source = <ISerializeDataSource>getContextSource(action.context);
+
+    const textStream = streamifyString(<string> source.value);
```

Comment:
So it seems like the CI fails because of the dependency on `streamify-string`, which is not web-safe.
Could you remove the dependency on `streamify-string` in this package, and revert to the solution you had before?